app/uk/gov/hmrc/pensionschemereturn/transformations/nonsipp/SharesToEtmp.scala

```diff
@@ -51,7 +51,8 @@ class SharesToEtmp @Inject() extends Transformer {
       unquotedSharesWereDisposed = transformToSharesDisposalFlag(optShareTransactions, TypeOfShares.Unquoted),
       connectedPartySharesWereDisposed =
         transformToSharesDisposalFlag(optShareTransactions, TypeOfShares.ConnectedParty),
-      shareTransactions = optShareTransactions.map(_.map(transformShareTransactions)),
+      shareTransactions =
```

Comment:
optShareTransactions is already optional and if it is none, transformShareTransactions wouldn't be called. I couldn't figure out what this new change doing here differently ?

Reply:
Before this change, an empty list was passed in to ETMP and that caused this exception:
```
uk.gov.hmrc.pensionschemereturn.models.PensionSchemeReturnValidationFailureException: Invalid payload when submitStandardPsr :-
$.shares.shareTransactions: there must be a minimum of 1 items in the array
```
as the shareTransactions minItems property in schema is 1

Reply:
I've had a look and it must have been a frontend change that fixed the issue, as this new code does produce the same result as the original so I'm not sure now why it looked like it did something different whilst debugging. I'm happy to close this provided the FE changes still work without it.